packages/auth-server/package.json

```diff
@@ -17,6 +17,9 @@
     "@vueuse/nuxt": "^11.1.0",
     "@wagmi/core": "^2.13.3",
     "@wagmi/vue": "^0.0.49",
+    "axios": "^1.7.9",
```

Comment:
What do you think about just using `fetch`? I believe we are not doing anything fancy with the requests, It's basically a GET request. I believe we can avoid adding a dependency here.